src/translations/installer_ru.ts

```diff
@@ -0,0 +1,244 @@
+        <source>UninstallFailed</source>
+        <translation>${PRODUCT_NAME} ${VERSION} не был полностью удалён.$\r$\nПожалуйста, перезапустите Windows и снова запустите программу удаления.</translation>
+    </message>
+    <message>
+        <source>FileDeleteErrorDialog</source>
+        <translation>Ошибка удаления файла:$\r$\n$\r$\n$0$\r$\n$\r$\nНажмите Повторить, чтоб продолжить, или$\r$\nОтменить для отмены удаления.</translation>
+    </message>
+    <message>
+        <source>FileDeleteError</source>
+        <translation>Ошибка удаления файла $0</translation>
+    </message>
+    <message>
+        <source>Install64On32BitWarning</source>
+        <translation>Вы устанавливаете 32-разрядную версию ${PRODUCT_NAME} на Вашей 64-разрядной Windows. Доступна 64-разрядная версия ${PRODUCT_NAME} для загрузки.</translation>
+    </message>
+    <message>
+        <source>Install32On64BitError</source>
+        <translation>Установщик содержит 64-разрядную версию ${PRODUCT_NAME}. На Вашем компьютере устновлена 32-разрядная версия Windows, которая не может запускать 64-разрядные программы. Пожалуйста, загрузите 32-разрядныйустановщик ${PRODUCT_NAME}.</translation>
+    </message>
+    <message>
+        <source>UnsupportedWindowsVersionError</source>
+        <translation>Эта программа требует как минимум Windows ${MIN_WINDOWS_VER}.</translation>
+    </message>
+    <message>
+        <source>UninstallerAborted</source>
+        <translation>Удаление ${PRODUCT_NAME} было прервано. Пожалуйста, попробуйте позже.</translation>
+    </message>
+    <message>
+        <source>SettingsLink</source>
+        <translation>${PRODUCT_NAME} настройка</translation>
+    </message>
+    <message>
+        <source>UninstallRestartAsUserQuestion</source>
+        <translation>Деинсталлятор запускается с правами администратора, но для администратора, похоже, нет установки ${PRODUCT_NAME}. Вы хотите перезапустить программу удаления без прав администратора?</translation>
+    </message>
+    <message>
+        <source>OpenUrl</source>
+        <translation>Открытие $0 в браузере по умолчанию...</translation>
+    </message>
+    <message>
+        <source>MissingVcRuntime</source>
+        <translation>Не обнаружен Visual C++ Runtime ${ARCH}.</translation>
+    </message>
+    <message>
+        <source>MissingVcRuntime_Dialog</source>
+        <translation>На этом компьютере не найден следующий обязательный компонент:$\r$\nVisual C++ Runtime ${ARCH}$\r$\nВы хотите загрузить установщик для компонента?</translation>
+    </message>
+    <message>
+        <source>MissingVcRuntime_Retry</source>
+        <translation>После завершения загрузки, пожалуйста, запустите загруженный установщик. После завершения установки нажмите кнопку ОК, чтобы снова проверить состояние установки.</translation>
+    </message>
+    <message>
+        <source>MissingVcRuntime_Found</source>
+        <translation>Требуемый компонент уже установлен на вашем компьютере</translation>
```

Comment:
```suggestion
        <translation>Требуемый компонент уже установлен на вашем компьютере.</translation>
```
This on the other hand is a sentence that is displayed to the user and should have a dot at the end.

@gyunaev suggested a change here too, please have a look:

> Требуемый компонент был обнаружен на вашем компьютере
there is no need to use 'successfully' here in Russian, "Was found" implies success.